modules/nf-core/samtools/sormadup/main.nf

```diff
@@ -0,0 +1,99 @@
+process SAMTOOLS_SORMADUP {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.19.2--h50ea8bc_0' :
+        'biocontainers/samtools:1.19.2--h50ea8bc_0' }"
+
+    input:
+    tuple val(meta), path(input)
+    tuple val(meta2), path(fasta)
+
+    output:
+    tuple val(meta), path("*.{bam,cram}")   , emit: markdup
+    tuple val(meta), path("*.{bai,crai}")   , emit: markdup_index, optional: true
+    tuple val(meta), path("*.metrics")      , emit: metrics
+    path "versions.yml"                     , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args ?: ''
+    def args3 = task.ext.args ?: ''
+    def args4 = task.ext.args ?: ''
+    def args5 = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def extension = args.contains("--output-fmt sam") ? "sam" :
+                    args.contains("--output-fmt bam") ? "bam" :
+                    args.contains("--output-fmt cram") ? "cram" :
+                    "bam"
+    def reference = fasta ? "--reference ${fasta}" : ""
+    def sort_memory = (task.memory.mega/task.cpus*0.75).intValue()
```

Comment:
I'm aware it isn't ideal, but upping the memory in samtools sort speeds it up tremendously in our tests.
Would you be comfortable with a more conservative 60% instead of 75?

Reply:
I think I would integrate this module with a patch to remove the requirement. Then do some tests and feed back to you / reinstate the requirement.

Let's just keep this line for now